setup.py

```diff
@@ -151,7 +151,7 @@
     "psutil",
     "pyyaml>=5.1",
     "pydantic",
-    "pytest",
```

Comment:
min. version to make the following work (used in `src/transformers/testing_utils.py` for the doctest in PRs)
```python
from _pytest.doctest import Module,
```